docs/reference/changelog-r2020.md

```diff
@@ -30,6 +32,7 @@ Released on December YYth, 2019.
     - Linux: Added support for Python 3.8.
   - Enhancements
     - Improved the [Sick LD MRS](../guide/lidar-sensors.md#sick-ld-mrs) PROTO to support the following types: `400001`, `400102`, `400001S01`, `400102S01` and `800001S01`.
+    - **Improved the `wb_supervisor_simulation_reset` to avoid restarting the controllers.**
```

Comment:
Adding a remark about which function should be used instead would be better here.